packages/coinstac-ui/app/render/components/maps/fields/maps-csv-field.jsx

```diff
@@ -233,8 +233,10 @@ class MapsCsvField extends React.Component {
         return acc;
       }, {});
 
+      const mapColumns = map.map(elem => elem.column);
+
       this.setState(prevState => ({
-        remainingHeader: prevState.remainingHeader.filter(column => map.includes(column)),
```

Comment:
It looks like this filter didn't work to begin with because `map` is an array of objects and `.includes` is only checking for column names. Meaning it would always be false.